frontend/src/components/Button/Button.tsx

```diff
@@ -0,0 +1,43 @@
+import { ReactNode, MouseEvent } from 'react';
+import { StyledButton } from './Button.styled';
+import { NativeButtonProps } from 'antd/lib/button/button';
+import 'antd/dist/antd.css';
```

Comment:
To chcielibyśmy zaimportować w miejscu, które zawsze odpali się zarówno w projekcie jak i testach :) Znasz takie?
`<AppProviders>`